src/oneNoteDataStructures/notebook.ts

```diff
@@ -3,10 +3,14 @@ import { Section } from './section';
 import { SectionGroup } from './sectionGroup';
 
 export interface Notebook extends OneNoteItem {
+	// Properties that need to be loaded-on-demand
+	apiHttpErrorCode?: number;
+
 	expanded: boolean;
 	sectionGroups: SectionGroup[];
 	sections: Section[];
 	webUrl: string;
 	apiUrl?: string;
 	lastModifiedTime: Date;
+	needsToFetchChildren?: boolean
```

Comment:
I don't see this being set to true anywhere

Reply:
So we would never fetch sections?

Reply:
The getNotebooks error is caught by the app, so that is where this is being set. So you guys would need to set this to true in Teams. Let me know if you think of a better way to handle this.

Reply:
Yes - this should be expandable if the notebook.sections property is undefined, which is the case when we set non-expanded notebooks. That is better than the app setting this property in that case (the sections being undefined is the property that should trigger this, not the needsToFetchChildren property)